build-canal-resources.sh

```diff
@@ -28,7 +28,7 @@ git clone $FLANNEL_REPO "${canal_temp}/flannel"
 pushd ${canal_temp}/flannel
 git checkout "$FLANNEL_COMMIT"
 ARCH="$ARCH" ./build-flannel-resources.sh
-mv flannel-*.gz ${canal_root}
+mv flannel-*.tar.gz ${canal_root}
```

Comment:
I suppose you'll need to update the FLANNEL_COMMIT to `d71d81e42c021a30323786d5cad44825efcf42c4`